### PR TITLE
[f] documentation to resolve Bad Request (400)

### PR DESCRIPTION
**Pull request by egmen**

### CODE REVIEW

**Comment by egmen** [comment]
fix for #3

**Comment by ThomDietrich** [comment]
Hey @egmen,
Thanks! As we know from #3 this was overdue for a long time!

One kind request though. This is a known issue *today* but imho should be documented as a normal part of the setup process. Every user must take care of this, otherwise the Addon won't work.

Could you please mention this step under "TL;DR;" and right before "Remote Server Configuration", maybe under the headline "Local HA Network Configuration"? Thanks!

File: autossh/DOCS.md [hunk 1]
```diff
@@ -123,3 +123,14 @@ This is optional and for testing purposes a verbose output enabled by `-v` can b
 A key pair is generated when the container is first initialized in your environment.
 Set this to `true` if you even need to urge to regenerate a key.
 
+## Known issues
+
+### Error on public domain - 400: Bad Request
+
+Need to add to your `/config/configuration.yaml` this config (see https://www.home-assistant.io/integrations/http/#trusted_proxies):
```
Comment (ThomDietrich) [inline autossh/DOCS.md:130]:
```suggestion
Please add the following lines to your `configuration.yaml`. For a description of this security measure see: https://www.home-assistant.io/integrations/http/#trusted_proxies
```

File: autossh/DOCS.md [hunk 2]
```diff
@@ -123,3 +123,14 @@ This is optional and for testing purposes a verbose output enabled by `-v` can b
...
+```yaml
+http:
+  use_x_forwarded_for: true
+  trusted_proxies:
+    - 192.168.88.88 # Replace this with yours internal Home Assistant IP-address
```
Comment (ThomDietrich) [inline autossh/DOCS.md:135]: The number 88 has a bad reputation in my region 🙃

```suggestion
    - 192.168.0.2  # Replace with the internal IP address of your Home Assistant host
```

**Comment by ThomDietrich** [comment]
Thank you!!

**Pull request merged**